basis/db/sqlite/sqlite.factor

```diff
@@ -161,6 +161,7 @@ M: sqlite-db-connection persistent-table ( -- assoc )
         { +user-assigned-id+ { f f f } }
         { +random-id+ { "integer" "integer" f } }
         { +foreign-id+ { "integer" "integer" "references" } }
+        { +primary-key+ { f f "primary key" } }
```

Comment:
do you have a test case that shows this working where it failed before?

Reply:
Of course:
```
USING: db db.tuples db.types db.sqlite ;
TUPLE: t data ;
t "t" { { "data" "data" +primary-key+ } } define-persistent
"test.db" <sqlite-db> [ t ensure-table ] with-db
```

```
no-sql-type
type +primary-key+
To continue, pick one of the options below:
Abort
```